### DISCUSSION TITLE
disable fancybox carousel slideshow autoplay

### DISCUSSION THREAD

**Discussion by pjmdesi** [fancyapps/ui#393] [Q&A]
Hi there, I have been using fancybox on my site for a while. I recently implemented the fancyapps' carousel library with the autoplay plugin. It all works great.

The issue is the original fancybox components which open as galleries (multiple items in a slideshow) now autostart as a slideshow even for videos. I'd like to disable this by default.
There is no default to disable autoplay from what I can see, and all the info I could find online was for previous versions of fancybox.

Using the following code does not seem to work because I think the autoplay plugin is overriding the setting:
```
Fancybox.bind('[data-fancybox]', {
    slideshow: {
        autoStart: false
    }
});
```

Help is greatly appreciated.

**Comment by fancyapps** [comment]
Hi, Please, join the Discord server for chat - https://discord.gg/6QYFwv6Z

**Reply by Roman52** [reply]
Hello, @fancyapps, what is the answer for this? I have the same issue.
Thanks.

**Reply by fancyapps** [reply]
Well, you could try something like this (for Fancybox options): `{ Carousel : { Autoplay: false } }`, I think it should work, sorry, but I have not tried. Anyway, the best solution would be to upgrade to the upcoming v5 that will be available very soon.

**Reply by Roman52** [reply]
Thanks @fancyapps 
`{ Carousel : { Autoplay: false } }` is working.
